deploy/runtools/switch_model_config.py

```diff
@@ -166,17 +168,18 @@ def local_logged(command: str) -> None:
         local_logged("cd " + switchbuilddir + " && make")
         local_logged("mv " + switchbuilddir + "switch " + switchbuilddir + binaryname)
 
-    def get_switch_simulation_command(self, sudo: bool) -> str:
+    def get_switch_simulation_command(self) -> str:
         """ Return the command to boot the switch."""
         switchlatency = self.fsimswitchnode.switch_switching_latency
         linklatency = self.fsimswitchnode.switch_link_latency
         bandwidth = self.fsimswitchnode.switch_bandwidth
-        # insert gdb -ex run --args between sudo and ./ below to start switches in gdb
-        return """screen -S {} -d -m bash -c "script -f -c '{} ./{} {} {} {}' switchlog"; sleep 1""".format(self.switch_binary_name(), "sudo" if sudo else "", self.switch_binary_name(), linklatency, switchlatency, bandwidth)
+        # insert gdb -ex run --args in front of ./ below to start switches in gdb
+        return """screen -S {} -d -m bash -c "script -f -c './{} {} {} {}' switchlog"; sleep 1""".format(self.switch_binary_name(), self.switch_binary_name(), linklatency, switchlatency, bandwidth)
```

Comment:
Shouldn't we check for `is_on_aws()` and use `sudo` accordingly here as well?

Reply:
I was able to run a switch simulation without sudo on AWS so I don't think that's needed...